pkg/ratelimit/limitware/middleware.go

```diff
@@ -110,6 +111,10 @@ func NewMiddleware(ctx context.Context, s limiter.Store, f httplimit.KeyFunc, op
 		return nil, fmt.Errorf("stat view registration failure: %w", err)
 	}
 
+	if err := view.Register(redis.ObservabilityMetricViews...); err != nil {
```

Comment:
It shouldn't be an issue, but we may in the future want to move this into the main functions of the servers, but for now this is fine here.

Reply:
Noted.